app/src/main/java/com/helio/app/UserDataViewModel.java

```diff
@@ -32,10 +40,34 @@ public UserDataViewModel(@NonNull Application application) {
         return motors;
     }
 
+    public LiveData<Map<Integer, LightSensor>> fetchLightSensors() {
+        if (lightsensors == null) {
+            lightsensors = new MutableLiveData<>();
+            client.getAllLightSensors(lightsensors);
+        }
+        return lightsensors;
+    }
+
+    public LiveData<Map<Integer, MotionSensor>> fetchMotionSensors() {
+        if (motionsensors == null) {
+            motionsensors = new MutableLiveData<>();
+            client.getAllMotionSensors(motionsensors);
+        }
+        return motionsensors;
+    }
+//    public LiveData<Map<Integer, Sensor>> fetchSensors(){
```

Comment:
If we don't want to use it, it should be removed rather than commented. Just to keep the `main` branch clean